deployment/helm/Chart.yaml

```diff
@@ -5,7 +5,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 0.0.0
+version: 0.0.1
```

Comment:
I guess the problem is the Helm tool won't write the versioning (and won't compose the docker container image correctly) unless the variables are set in Chart.yaml?  Helm isn't reading VERSION_CHART.txt

I synchronized the versions in Chart.yaml with the VERSION_CHART/VERSION_APP.txt

Reply:
Although I see now you have a helm repo for this and its Chart.yaml is populated correctly.  Are you synchronizing those two on the fly (via argo or something similar) before posting it to the repo?

Reply:
Good guess! It's actually a matter of my GitHub workflow design for building artifacts. I have three of those:
- container image
- helm chart
- starlight based docs

Since 3 of those utilize notion of app version and last 2 need info about chart version, it seemed sensible for me to "factor it out" into one source of truth. In this case as two text files in root directory of the project.

Nevertheless, I do see now how this, can be counter intuitive for Helm Chart developers... Thank you for pointing this out!

I'm seeking for advice: would you rather see some additional documentations about this on projects website, or would it be a better idea to move "source of truth" for Helm Chart version into `Chart.yaml` itself?

Reply:
I'm a fan of technical stuff "just working" which for me, would imply putting it in Chart.yaml in addition to mentioning this in your docs.

But also- add comments to Chart.yaml indicating how this is being used.  It's leaving breadcrumbs for folks who encounter your project (or even find it on some other public helm chart artifact site) and gives them a link back to your original docs.